umh-core/vendor/go.yaml.in/yaml/v3/README.md

```
@@ -0,0 +1,171 @@
+  Otherwise they behave as a string.
+  Booleans in YAML 1.2 are `true`/`false` only.
+* Supports octals encoded and decoded as `0777` per YAML 1.1, rather than
+  `0o777` as specified in YAML 1.2, because most parsers still use the old
+  format.
+  Octals in the `0o777` format are supported though, so new files work.
+* Does not support base-60 floats.
+  These are gone from YAML 1.2, and were actually never supported by this
+  package as it's clearly a poor choice.
+
+
+## Installation and Usage
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)